build/__tests__/post-build/v1/press.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs-extra')
+const csv = require('csv-parser')
+const checkFile = require('../../utitlities/check-file')
+
+describe('File : Press', () => {
+  it('exists', () => {
+    checkFile('_data/v1/press')
+  })
+
+  it('matches field pattern', done => {
+    fs.createReadStream('_data/v1/press.csv')
+      .pipe(csv())
+      .on('data', row => {
+        expect(row).toEqual(
+          expect.objectContaining({
+            title: expect.anything(),
```

Comment:
(blanket question) Is there any reason to use `expect.anything()` instead of more specific tests like `.toBe('string')`?

At the very least, we should probably validate things that could break down the line (urls, dates, numbers).

Reply:
Right now because the JSON files don't list values if there is nothing (i.e. if the grade is empty, then instead of `"grade":null` we get no grade value at all) we are using the CSV files for testing. It is a little janky, but it works.

However, CSV files turn everything into a string or null value.

Reply:
Ah, right. Hopefully we can be more comprehensive about that down the line, then.